bugbug/models/stepstoreproduce.py

```diff
@@ -58,25 +57,20 @@ def __init__(self, lemmatization=False):
             ]
         )
 
-        self.clf = xgboost.XGBClassifier(n_jobs=16)
-        self.clf.set_params(predictor="cpu_predictor")
+        self.clf = PUClassifier(n_jobs=-1)
 
     def get_labels(self):
         classes = {}
 
         for bug_data in bugzilla.get_bugs():
+            classes[int(bug_data["id"])] = 0
+            for entry in bug_data["history"]:
+                for change in entry["changes"]:
+                    if change["removed"].startswith("stepswanted"):
+                        classes[int(bug_data["id"])] = 1
             if "cf_has_str" in bug_data:
-                if bug_data["cf_has_str"] == "no":
-                    classes[int(bug_data["id"])] = 0
-                elif bug_data["cf_has_str"] == "yes":
+                if bug_data["cf_has_str"] == "yes":
                     classes[int(bug_data["id"])] = 1
-            elif "stepswanted" in bug_data["keywords"]:
-                classes[int(bug_data["id"])] = 0
-            else:
-                for entry in bug_data["history"]:
-                    for change in entry["changes"]:
-                        if change["removed"].startswith("stepswanted"):
-                            classes[int(bug_data["id"])] = 1
 
```

Comment:
Instead of modifying everything here, you can just keep it as it was before and then do:

```Python
if int(bug_data["id"]) not in classes:
    classes[int(bug_data["id"])] = 0
```